### ISSUE TITLE
[BUG] when trying to run function QueueTrigger based

### ISSUE THREAD

**Issue by thdotnet** [Azure/azure-functions-host#4847]
I am trying to run my Azure Function project using Storage Emulator, but I'm getting the following error:

System.MethodAccessException: 'Attempt by method 'Microsoft.Azure.WebJobs.Host.Queues.Listeners.QueueListener+<ExecuteAsync>d__27.MoveNext()' to access method 'Microsoft.Azure.WebJobs.Host.Protocols.FunctionDescriptor.get_LogName()' failed.'

It's a queue trigger, pointing to storage emulator.

**Comment by thdotnet** [comment]
@brettsam  ?

**Comment by brettsam** [comment]
What does it say when you start up the CLI?

For example...
```
Azure Functions Core Tools (2.7.1585 Commit hash: 9a2b7239b937148e3b76e6f7999798629a82134a)
Function Runtime Version: 2.0.12641.0
```

**Comment by thdotnet** [comment]
@brettsam at this moment if I try to run, it will download the latest version. Any way, try to reproduce with the information I've provided above. I'll let it upgrade and see if the problem is solved while that

**Comment by brettsam** [comment]
Functions VS Tools 2.32 -> Functions Core Tools 2.7.1575 -> Functions Host 2.0.12641, which should work. The `LogName` property was added the build before this. The latest storage extension uses that property, so if the host version you're running is earlier, it won't be found.

Azure Functions CLI - 1.0.12, I believe is for v1 only won't work (it's a year old).

The best way to tell what's you're using is via the startup output line above. We've got a lot of versions of different pieces and it can be hard to piece together what's actually running sometimes.

Using the latest VS tools and hitting F5 with the latest storage extension has been working for me.
